### DISCUSSION TITLE
Date / time picker in firefox broken?

### DISCUSSION THREAD

**Discussion by Kimotu** [beercss/beercss#280] [Q&A]
Hello,

I am playing around with beerCSS to check whether to use it for future projects or migration. So I created a small web page that uses date picker inputs. But it seems that date picker (time also) does not work in firefox. Is there a workaround?

I opened the component [example page](https://www.beercss.com/), scrolled to _custom inputs_ which uses this piece of code

`<button class="circle">
  <i>today</i>
  <input type="date">
`</button>``
`<button class="circle">
  <i>schedule</i>
  <input type="time">
</button>`

- Firefox does not work (124.0.1)
- Mobile Firefox works
- Opera works
- Edge works
- Chromium works

Can anybody confirm this problem or know how to resolve it?

**Edit:** I clicked more dozent times on the date picker and there is a very small area in the right,bottom of the calendar symbol where the date picker opens. But you have to know it and target the correct spot. Same with the labeled date picker. There you have to hit an area behind the letter 'e'. But I still did not find a point to trigger the time picker. I think there is some kind of overlay that blocks the click.

**Comment by leonardorafael** [comment]
It seems Firefox doesn't show a time picker on desktop version by default (there are nothing to do here).

About the icon of date picker, there are some different behavior to style this on Firefox to get all area of container like other browsers. (needs to check how to do that)

**Reply by Kimotu** [reply]
Well, the time picker indeed is not such important, but a working datepicker is essential for my project. Maybe you can implement a helper that shows the date without the need to open the datepicker like the css I am currently using.

![grafik](https://github.com/beercss/beercss/assets/45389306/a7112334-e487-44d9-9854-c6017cc4c0e4)
